crates/testing/src/block_types.rs

```diff
@@ -83,21 +81,32 @@ impl VIDBlockPayload {
     pub fn genesis() -> Self {
         let txns: Vec<u8> = vec![0];
         // It's impossible for `encode` to fail because the transaciton length is very small.
-        let encoded = VIDTransaction::encode(vec![VIDTransaction(txns.clone())]).unwrap();
-        VIDBlockPayload {
-            transactions: vec![VIDTransaction(txns)],
+        let encoded = TestTransaction::encode(vec![TestTransaction(txns.clone())]).unwrap();
+        TestBlockPayload {
+            transactions: vec![TestTransaction(txns)],
             payload_commitment: vid_commitment(&encoded),
         }
     }
 }
 
-impl Display for VIDBlockPayload {
+impl Committable for TestBlockPayload {
```

Comment:
Is this needed? I think we no longer need `Committable` for block payload. Maybe this was added due to some merge issue.

Reply:
Oh, git sneaked that in while rebasing I think. Thank you for the catch!